src/status_im/ui/screens/accounts/recover/views.cljs

```diff
@@ -32,7 +32,7 @@
       :multiline           true
       :default-value       passphrase
       :auto-correct        false
-      :on-change-text      #(re-frame/dispatch [:set-in [:accounts/recover :passphrase] %])
+      :on-change-text      #(re-frame/dispatch [:set-in [:accounts/recover :passphrase] (string/lower-case %)])
```

Comment:
How about preventing upper case directly? Does this convert the display too?

Reply:
@jeluard If you type say `D` it immediately turns into `d` before you even type the next letter. So, yes - this is both value and display.

Reply:
@dmitryn Not sure that would work as expected in a text input.

Reply:
@goranjovic yes, just checked and it doesn't work. And your solution is working fine for me, auto corrects text in place.

Reply:
TBH, I would've preferred something like a lowercase-only keyboard. But it wouldn't really solve anything since a lot of people would probably copy paste it, not actually type.